Trimble.Modus.Components/Controls/TMInput/CustomInput.xaml.cs

```diff
@@ -0,0 +1,551 @@
+using System.Windows.Input;
+using Trimble.Modus.Components.Constant;
+using Trimble.Modus.Components.Enums;
+using Trimble.Modus.Components.Helpers;
+
+namespace Trimble.Modus.Components.Controls;
+
+public partial class CustomInput : ContentView
+{
+
+    #region Private Properties
+
+    private const double disabledOpacity = 0.4;
+
+    private ValidationResponse _validationResponse;
+    private CustomInput _customInput;
+
+    #endregion
+
+    #region Bindable Properties
+
+    /// <summary>
+    /// Identifies the TextProperty bindable property. This property is used to display the property text
+    /// </summary>
+    public static readonly BindableProperty TextProperty =
+       BindableProperty.Create(nameof(Text), typeof(string), typeof(CustomInput), propertyChanged: OnTextChanged);
+
+    /// <summary>
+    /// Identifies the TextColor bindable property. This property is used to define the color of the entry text
+    /// </summary>
+    public static readonly BindableProperty TextColorProperty =
```

Comment:
The user cannot change the TextColor of the Input. Do we have to expose this property now?

Reply:
Same question for the Font and FontSize properties in this class.

Reply:
It was already in TMInput so I had added the same to this

Reply:
Okay Removed those properties